docs/src/main/sphinx/security/internal-communication.md

```diff
@@ -108,6 +108,10 @@ configuration properties:
 - `internal-communication.https.keystore.path`
 - `internal-communication.https.truststore.path`
 - `node.internal-address-source`
+
+Also, note that you need to configure `http-server.https.automatic-shared-secret`
```

Comment:
Why not add this to point 4? Or 2 .. or both? .. does this really belong in this note?